### PR TITLE
HPCC-14060 Remove legacy largeMemSize

### PR DESCRIPTION
**Pull request by jakesmith**
Signed-off-by: Jake Smith jake.smith@lexisnexis.com

### CODE REVIEW

**Comment by jakesmith** [comment]
NB: 1st commit is unmerged hpcc-14059 PR.
@richardkchapman - please review

**Comment by jakesmith** [comment]
@richardkchapman - rebased

File: thorlcr/graph/thgraph.cpp [hunk 1]
```diff
@@ -2516,17 +2516,13 @@ void CJobBase::init()
     thorAllocator.setown(createThorAllocator(((memsize_t)globalMemorySize)*0x100000, memorySpillAt, *logctx, crcChecking, usePackedAllocator));
 
     unsigned defaultMemMB = globalMemorySize*3/4;
-    unsigned largeMemSize = getOptUInt("largeMemSize", defaultMemMB);
```
Comment (richardkchapman) [inline thorlcr/graph/thgraph.cpp]: Has largeMemsize been removed from the relevant xsd files, default environment.xml, documentation etc?

Comment (jakesmith) [inline thorlcr/graph/thgraph.cpp]: Must have previously been removed. Not in xsd/xslt's , environment(s) or docs.
@richardkchapman

**Pull request merged**